.ci/Jenkinsfile

```
@@ -343,6 +345,19 @@ pipeline {
             }
           }
         }
+        stage('Publish to npm') {
+          steps {
+            withNodeJSEnv(version: 'v14.17.5') {
+              withNpmrc(secret: "${env.NPMRC_SECRET}", path: "${env.WORKSPACE}/${env.BASE_DIR}") {
+                withTotpVault(secret: "${env.TOTP_SECRET}", code_var_name: 'TOTP_CODE') {
+                  dir("${BASE_DIR}") {
+                    sh(label: 'npm-publish', script: 'npm publish --otp ${TOTP_CODE}')
```

I notice when comparing to https://github.com/elastic/synthetics/blob/9019b2b03b91170afa2d54cea25d790f601ffe56/.ci/Jenkinsfile#L194-L196 that that uses `env.TOTP_CODE` rather than bare `TOTP_CODE`. Do you know if this needs to change to the former?

```suggestion
                    sh(label: 'npm-publish', script: 'npm publish --otp="${env.TOTP_CODE}"')
```

---

That's a cosmetic preference, I like to use `env.VARIABLE` to help with what's the variable coming from, while `VARIABLE`, it might  happen from either a groovy variable or an environment variable. No strong opinion, but the context about what's the `env` map coming from.